tests/test_tools.py

```diff
@@ -15,3 +16,20 @@ def test_get_submission_invitations(self):
     def test_get_all_venues(self):
         venues = openreview.get_all_venues(self.client)
         assert venues, "Venues could not be retrieved"
+
+    def test_iterget(self):
+        data_size = 1000
```

Comment:
I would write the test doing a get against the server that is now dev.openreview.net.